test/solium/no-arg-overlow-rule.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+let fs = require('fs');
+let dedent = require("dedent");
+let Solium = require("solium");
+
+let userConfig = JSON.parse(fs.readFileSync('.soliumrc.json', 'utf8'));
```

Comment:
Why `let` instead of `const`? Do we have (should we have) a rule for this?

Reply:
I've proposed #9 to add this rule.

Now this brings a super good topic, we were not running the linter on this code. I've updated the branch to do so.
TODO for later: add tests for the eslint rules.